pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -347,6 +347,24 @@ func (l *Launcher) FinalizeVirtualMachineMigration(_ context.Context, request *c
 	return response, nil
 }
 
+func (l *Launcher) HotplugHostDevices(_ context.Context, request *cmdv1.VMIRequest) (*cmdv1.Response, error) {
+	vmi, response := getVMIFromRequest(request.Vmi)
+	if !response.Success {
+		return response, nil
+	}
+
+	if err := l.domainManager.HotplugHostDevices(vmi); err != nil {
+		log.Log.Object(vmi).Errorf(err.Error())
+		response.Success = false
+		response.Message = getErrorMessage(err)
+		return response, nil
+	}
+
+	log.Log.V(3).Object(vmi).Info("hot-plug host-devices finished")
```

Comment:
This is confusing, the operation did not really finished.

Reply:
I removed this log message